apps/cowswap-frontend/src/modules/permit/const.ts

```diff
@@ -21,3 +23,9 @@ export const DEFAULT_PERMIT_GAS_LIMIT = '80000'
 export const DEFAULT_PERMIT_VALUE = MaxUint256.toString()
 
 export const DEFAULT_PERMIT_DURATION = ms`5 years`
+
+export const ORDER_TYPE_SUPPORTS_PERMIT: Record<TradeType, boolean> = {
+  [TradeType.SWAP]: true,
+  [TradeType.LIMIT_ORDER]: false,
+  [TradeType.ADVANCED_ORDERS]: false,
+}
```

Comment:
We can use a feature flag for this, but since we expect limit orders will be supported soon, just a simple const for now